src/cursor/abstract_cursor.ts

```diff
@@ -642,6 +642,8 @@ export abstract class AbstractCursor<
           this[kId] =
             typeof response.cursor.id === 'number'
               ? Long.fromNumber(response.cursor.id)
```

Comment:
Do we actually test this anywhere?  We don't enable `useBigInt64` in our tests, so we're not getting implicit coverage there.

The only place that I see some coverage is in the change streams tests this PR adds.

Can we add tests to confirm that cursors properly handle `useBigInt64: true`?  I think we can follow the example set in `integration/abstract_cursor.test.ts` (which uses a find cursor to test generic functionality).